src/resources/payment_source.rs

```diff
@@ -11,51 +11,23 @@ use serde_derive::{Deserialize, Serialize};
 /// Not to be confused with `SourceParams` which is used by `Source::create`
 /// to create a source that is not necessarily attached to a customer.
 #[derive(Clone, Debug)]
-pub enum PaymentSourceParams<'a> {
+pub enum PaymentSourceParams {
     /// Creates a payment method (e.g. card or bank account) from tokenized data,
     /// using a token typically received from Stripe Elements.
     Token(TokenId),
 
     /// Attach an existing source to an existing customer or
     /// create a new customer from an existing source.
     Source(SourceId),
-
-    /// Creates a `Card` payment method from the full card information
-    /// (e.g. card number, expiration, etc).
-    ///
-    /// You usually want to use a `TokenId` received from
-    /// [Stripe Elements](https://stripe.com/docs/stripe-js)
-    /// instead.
-    Card(CardParams<'a>),
-
-    /// Creates a `BankAccount` payment method from the full bank information
-    /// (e.g. account number, expiration, etc).
-    ///
-    /// You usually want to use a `TokenId` received from
-    /// [Stripe Elements](https://stripe.com/docs/stripe-js)
-    /// instead.
-    BankAccount(BankAccountParams<'a>),
 }
 
-impl<'de> ::serde::Deserialize<'de> for PaymentSourceParams<'de> {
+impl<'de> ::serde::Deserialize<'de> for PaymentSourceParams {
```

Comment:
As with serialize, we can probably just use builtin serde derive instead now.

Reply:
updated here, too